README.md.jinja

```diff
@@ -5,6 +5,12 @@
 [![pipeline status]({{ gitlab_url }}/badges/{{ odoo_version }}/pipeline.svg)]({{ gitlab_url }}/commits/{{ odoo_version }})
 [![coverage report]({{ gitlab_url }}/badges/{{ odoo_version }}/coverage.svg)]({{ gitlab_url }}/commits/{{ odoo_version }})
 {%- endif %}
+{%- if domains_prod %}
+[![Deployment data](https://img.shields.io/badge/Deployment-prod-green)](https://{{ domains_prod[0].hosts[0] }})
+{%- endif %}
+{%- if domains_test %}
+[![Deployment data](https://img.shields.io/badge/Deployment-demo-yellow)](https://{{ domains_test[0].hosts[0] }})
```

Comment:
Same as above, but for test:

https://github.com/Tecnativa/doodba-copier-template/blob/788851afcd758e837f266867eea51a8cd46f40b2/test.yaml.jinja#L32

https://github.com/Tecnativa/doodba-copier-template/blob/788851afcd758e837f266867eea51a8cd46f40b2/copier.yml#L281-L283